app/models/api_request.rb

```diff
@@ -6,10 +6,16 @@ def response
   private
 
   def format_response(res)
+    response_body = begin
+      JSON.parse(res.body)
+    rescue JSON::ParserError
```

Comment:
[question] is this just a general guard or did you see something specific that would make this happen?